web/client/epics/__tests__/wfsquery-test.js

```diff
@@ -347,37 +384,46 @@ describe('wfsquery Epics', () => {
                 });
                 done();
             },
-            {
-                query: {
-                    data: {},
-                    featureTypes: [],
-                    typeName: 'layer1',
-                    url: '/dummy'
-                },
-                featuregrid: {
-                    timeSync: true,
-                    pagination: {
-                        size: 10
-                    },
-                    open: true,
-                    selectedLayer: "layer1",
-                    changes: [],
-                    mode: 'VIEW'
-                },
-                layers: {
-                    flat: flatLayers,
-                    layerMetadata: {
-                        expanded: false,
-                        maskLoading: false
-                    },
-                    settings: {
-                        expanded: false,
-                        node: null,
-                        nodeType: null,
-                        options: {}
+            mockState
+            );
+        });
+
+        it('featureTypeSelectedEpic', (done) => {
+            mockState.layers.flat = wmsLayer;
```

Comment:
Changing test data on the fly is dangerous for maintainance. Please do a copy of it.